### PR TITLE
added feature flag info for pgbouncer strategy

### PR DESCRIPTION
**Pull request by rishkarajgi**
Added feature flag information for the new pgbouncer calculation strategy feature in 0.33.0

### CODE REVIEW

File: software/release-notes.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ Release date: August 3, 2023
 ### Automatic PGBouncer connection scaling
 
 Astronomer Software now automatically scales the size of PGBouncer connection pools based on your Airflow component counts and Airflow configuration, instead of solely based on total AU. This improves performance, scalability, and utilization of database connections across all Deployments. 
+You can enable this by setting `deployments.pgBouncerResourceCalculationStrategy: airflowStratV2`
```
Comment (jwitz) [inline software/release-notes.md:21]:
```suggestion

This feature is off by default. You can enable it by setting  `deployments.pgBouncerResourceCalculationStrategy: airflowStratV2` in your `config.yaml` file. To revert back to previous behavior, set this key to `auStratV1` instead.
```

File: software/release-notes.md [hunk 2]
```diff
@@ -18,6 +18,11 @@ Release date: August 3, 2023
...
+
+:::info
```
Comment (jwitz) [inline software/release-notes.md:23]: We can delete this info note

**Review by jwitz — APPROVED** [review]

File: software/release-notes.md [hunk 3]
```diff
@@ -19,6 +19,12 @@ Release date: August 3, 2023
 
 Astronomer Software now automatically scales the size of PGBouncer connection pools based on your Airflow component counts and Airflow configuration, instead of solely based on total AU. This improves performance, scalability, and utilization of database connections across all Deployments. 
 
+This feature is off by default. You can enable it by setting  `deployments.pgBouncerResourceCalculationStrategy: airflowStratV2` in your `config.yaml` file. To revert back to previous behavior, set this key to `auStratV1` instead.
+
+:::info
+You can revert back to the AU based strategy by setting the value to `deployments.pgBouncerResourceCalculationStrategy: auStratV1`
```
Comment (jwitz) [inline software/release-notes.md:25]:
```suggestion
```

File: software/release-notes.md [hunk 4]
```diff
@@ -19,6 +19,12 @@ Release date: August 3, 2023
 
 Astronomer Software now automatically scales the size of PGBouncer connection pools based on your Airflow component counts and Airflow configuration, instead of solely based on total AU. This improves performance, scalability, and utilization of database connections across all Deployments. 
 
+This feature is off by default. You can enable it by setting  `deployments.pgBouncerResourceCalculationStrategy: airflowStratV2` in your `config.yaml` file. To revert back to previous behavior, set this key to `auStratV1` instead.
+
+:::info
```
Comment (jwitz) [inline software/release-notes.md:24]:
```suggestion
```

File: software/release-notes.md [hunk 5]
```diff
@@ -19,6 +19,12 @@ Release date: August 3, 2023
...
+You can revert back to the AU based strategy by setting the value to `deployments.pgBouncerResourceCalculationStrategy: auStratV1`
+:::
```
Comment (jwitz) [inline software/release-notes.md:26]:
```suggestion
```

File: software/release-notes.md [hunk 6]
```diff
@@ -19,6 +19,10 @@ Release date: August 3, 2023
 
 Astronomer Software now automatically scales the size of PGBouncer connection pools based on your Airflow component counts and Airflow configuration, instead of solely based on total AU. This improves performance, scalability, and utilization of database connections across all Deployments. 
 
+This feature is off by default. You can enable it by setting  `deployments.pgBouncerResourceCalculationStrategy: airflowStratV2` in your `config.yaml` file. To revert back to previous behavior, set this key to `auStratV1` instead.
+
+:::
```
Comment (jwitz) [inline software/release-notes.md:24]:
```suggestion
```

**Review by rujhan-arora-astronomer — APPROVED** [review]

**Pull request merged**